pages/validators/running_a_validator.md

```diff
@@ -0,0 +1,25 @@
+# Running a Validator
```

Comment:
given that this is a new page, I think we need to add this page to the metadata

https://github.com/dydxprotocol/v4-documentation/blob/6b273bfdd11a1cec6efdde0bc572bb92313a3d7f/pages/validators/_meta.json#L2